website/vercel.json

```diff
@@ -31,6 +31,11 @@
       "destination": "https://gh.getunleash.io/unleash-client-python/:path",
       "permanent": true
     },
+    {
+      "source": "/unleash-python-sdk(/?)/:path",
+      "destination": "https://gh.getunleash.io/unleash-python-sdk/:path",
+      "permanent": true
+    },
```

Comment:
Not sure why this is needed... but we had the same redirect above for the old python path, so I thought we might need it as well here